cmd/oci-image-tool/validate.go

```diff
@@ -67,6 +68,11 @@ func newValidateCmd(stdout, stderr *log.Logger) *cobra.Command {
 		`A set of refs pointing to the manifests to be validated. Each reference must be present in the "refs" subdirectory of the image. Only applicable if type is image or imageLayout.`,
 	)
 
+	cmd.Flags().BoolVarP(
+		&v.verbose, "verbose", "v", false,
+		fmt.Sprintf(`Output verbosely, including all objects digest to be validated. If unset, oci-image-tool will only output a OK or failed result.`),
+	)
```

Comment:
On Fri, Sep 09, 2016 at 02:43:03PM -0700, Stephen Day wrote:
> … while an increase in verbosity would provide richer test output.

What is “richer test output”?  Shouldn't we always run as many tests
as we have code for and leave it to TAP harnesses and such to distill
the data (if the user wants it distilled) [1]?

[1]: https://github.com/opencontainers/image-spec/pull/234#issuecomment-245949723